From 26a47f998c4884df7ba96a0f3ad6b268a832ae10 Mon Sep 17 00:00:00 2001
From: Michel Alexandre Salim <michel@michel-slm.name>
Date: Mon, 31 Aug 2020 18:11:02 -0700
Subject: [PATCH] Patch FindLua to support locating Lua 5.4

Lua 5.4 is out, and will be shipped with Fedora 33.

Since there are architectures where LuaJIT is not available, being able
to compile against Lua 5.4 is needed to get Neovim to build for this
release.

Signed-off-by: Michel Alexandre Salim <michel@michel-slm.name>
---
 cmake/FindLua.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/FindLua.cmake b/cmake/FindLua.cmake
index b669a49f2..7ba13e1f5 100644
--- a/cmake/FindLua.cmake
+++ b/cmake/FindLua.cmake
@@ -42,7 +42,7 @@ unset(_lua_append_versions)
 
 # this is a function only to have all the variables inside go away automatically
 function(_lua_set_version_vars)
-    set(LUA_VERSIONS5 5.3 5.2 5.1 5.0)
+    set(LUA_VERSIONS5 5.4 5.3 5.2 5.1 5.0)
 
     if (Lua_FIND_VERSION_EXACT)
         if (Lua_FIND_VERSION_COUNT GREATER 1)
-- 
2.26.2

